packages/frontity-org-theme/src/index.ts

```diff
@@ -45,6 +50,7 @@ const frontityOrg: FrontityOrg = {
         white: "#ffffff",
       },
       templates: ["fixed-header", "header", "footer", "newsletter"],
+      tabNumber: 1, // Used in the frontity-flow section of the homepage
```

Comment:
I would use here a more descriptive name (something with `flowSection`, maybe?).

Reply:
yeah, I thought the same, actually 😅 I used `activeFlowSectionTab` which is a bit verbose but it's better, because the "section" alone refers to the section of the website like "frontity flow" section or "headless flow" section, etc.